spec.html

```diff
@@ -23527,7 +23527,8 @@
     <!-- es6num="19.3.3" -->
     <emu-clause id="sec-properties-of-the-boolean-prototype-object">
       <h1>Properties of the Boolean Prototype Object</h1>
-      <p>The Boolean prototype object is the intrinsic object %BooleanPrototype%. The Boolean prototype object is an ordinary object. It is not a Boolean instance and does not have a [[BooleanData]] internal slot.</p>
```

Comment:
the term, "Boolean object" is defined in [4.3.16](http://ecma-international.org/ecma-262/6.0/#sec-boolean-object).

Generally, in the ES<=6 specs, a phrase like "a \<construct name> object" means the same thing as "a \<constructor name> instance".  In other words, an instance of the named constructor.